services/models.py

```diff
@@ -74,3 +84,16 @@ def __str__(self):
         },
         {"name": "created_at", "accessor": lambda x: x.strftime("%Y-%m-%d")},
     )
+
+    def download_gdpr_data(self):
+        """Download service specific GDPR data by profile
+        """
+        if self.service.gdpr_url:
+            url = urllib.parse.urljoin(self.service.gdpr_url, str(self.profile.pk))
+            try:
+                response = requests.get(url, timeout=5)
+                response.raise_for_status()
+                return response
+            except requests.exceptions.RequestException:
+                return response
```

Comment:
In case there's an exception should we return `{}` instead? :thinking:

Reply:
True dat